openmp/runtime/src/kmp_affinity.cpp

```diff
@@ -127,8 +127,9 @@ const char *__kmp_hw_get_catalog_string(kmp_hw_t type, bool plural) {
     return ((plural) ? KMP_I18N_STR(Threads) : KMP_I18N_STR(Thread));
   case KMP_HW_PROC_GROUP:
     return ((plural) ? KMP_I18N_STR(ProcGroups) : KMP_I18N_STR(ProcGroup));
+  default:
```

Comment:
In this and the other instances, is it just one or two cases missing for the switch to be covered, or what's the situation?

In general it's nicer to cover all cases explicitly, so that clang will warn if a new enumerator gets added later. But if they're too many, I suppose default is good too.

Reply:
I remove the `default` and explicitely covered all cases like you suggested